README.md

```diff
@@ -148,6 +148,56 @@ ioBroker has the ability to include files written by adapters in its backups. To
 
 This path is relative to the path returned by `getAbsoluteDefaultDataDir()`. The placeholder `%INSTANCE%` is automatically replaced by the instance number of each adapter, for example `"dataFolder": "my-adapter.%INSTANCE%"`.
 
+## OAuth2 token refresher
+To keep the OAuth2 access token up to date, you can use the `TokenRefresher` class.
+
+```Typescript
+import { TokenRefresher } from './lib/TokenRefresher';
```

Comment:
This is likely not too helpful for a developer that uses adapter-core package. so please adjust to whereever it is exported